.env.example

```diff
@@ -12,3 +12,6 @@ VERCEL_PROJECT_ID=""
 VERCEL_TOKEN=""
 GITHUB_TOKEN=""
 BASE_URL="http://localhost:3000"
+GATEWAY_SWAP_URL=<GATEWAY URL TO SUBMIT SWAP>
```

Comment:
hey, could not understand what do you plan to put here, because in another file https://github.com/darklakefi/dex-web/pull/98/files#diff-a964bb4ba6cfefe62c9d1f66ee2a328bdec55762fd6b7ef326d1edf048030d77R19 you use GATEWAY_HOST and GATEWAY_PORT not defined here